### ISSUE TITLE
Draft a README.md with repo basics, basedb hints

### ISSUE THREAD

**Issue by skylerbunny** [fuzzball-muck/fuzzball-muf#33]
Via some digging, I can see that several "considered-core" FB6 MUF files were eventually removed from here (and I would guess, from the basedb template at the same time). I surmise it's because their functions are present in FB7 so the MUFs are no longer needed. These include:

- `lib-gui.muf`
- `lib-index.muf`
- `lib-look.muf`

`lib-mpi.muf`, however, is still present in this repository but **not** in the basedb repo. Its latest commit comment suggests that at least at one time, it was present in the basedb.

The question is, is this oddity because `lib-mpi.muf` should have been deleted from this repo but wasn't, or because it should be in the Fuzzball basedb template, and isn't?

**Comment by wyld-sw** [comment]
This repo could certainly use some cleanup. The thinking at the time, I. believe, was that the programs within starterdb don't have a need for these libraries - but we can certainly include them if there's a use case.

**Comment by tanabi** [comment]
I believe it's acceptable for a MUF program to be in this repository and not in the basedb -- this repository (in my mind) includes any MUF that could be useful even if it isn't part of the starter DBs.

It's probably better lib-mpi isn't in FB7 by default because it has potential security issues (see https://github.com/fuzzball-muck/fuzzball/issues/619)

**Comment by skylerbunny** [comment]
Sounds good, that pretty much answers the question. What I found was the two year old README for FB6 that basically says 'You need to install the following libraries _in order_, and then you can install whatever you want', and the fate of this one was the only one I couldn't really work out. (The last commit it has that said that it's 'included with the basedb' confused some. I understand now; it no longer is, but that was purposely done.)

**Comment by tanabi** [comment]
Yeah, there's some updating to be done here.  And I still think we should fix lib-mpi just in case there's legacy programs that need it, but not including it in FB7 avoids a (albiet somewhat minor -- I don't know too many places that hand out MUCKER bits these days) security hole.  Let's use this issue as a reminder to fix the README :)

**Comment by skylerbunny** [comment]
> Let's use this issue as a reminder to fix the README :)

Ah, yeah, I guess you hit it on the head -- there isn't **any** README in the current commit of this repo that describes what to do with what's here, in terms of what you get in the basedb. I think there used to be, but it went away with the passage of time.

**Comment by wyld-sw** [comment]
I can put together a draft of a README, and possibly a deeper dive for some of the programs we have here. Some of the "must have these in order" mandate came from each program requiring `cmd-@register` (I have to remember to put @ commands in backticks...), which is now in-server - but there are other dependencies. Thanks for reminding us of this need!
